src/frontend/Pages/Dashboard.razor

```diff
@@ -0,0 +1,107 @@
+﻿@page "/Dashboard"
+@layout MainLayout
+
+
+<h1>Dashboard</h1>
+
+    <MudGrid Class="p-3">
+        <MudItem xs="12" sm="6" md="4">
+            <MudCard>
+                <MudCardContent>
+                    <MudChart ChartType="ChartType.Line" XAxisLabels="XAxisLabels" ChartSeries="Series" Height="250px" />
+                </MudCardContent>
+            </MudCard>
+        </MudItem>
+        <MudItem xs="12" sm="6" md="4">
+            <MudCard>
+                <MudCardContent>
+                    <MudChart ChartType="ChartType.Bar" Data="@barChartData" XAxisLabels="XAxisLabels" ChartSeries="Series" Height="250px" />
```

Comment:
Another MUD0002, I thinks it is InputData instead of just Data maybe?

Reply:
Auch dun